cmd/kpod/common.go

```diff
@@ -50,30 +78,59 @@ func getStore(c *cli.Context) (storage.Store, error) {
 	return store, nil
 }
 
+func getCopyOptions(reportWriter io.Writer, signaturePolicyPath string, srcDockerRegistry, destDockerRegistry *dockerRegistryOptions, signing signingOptions) *cp.Options {
+	if srcDockerRegistry == nil {
+		srcDockerRegistry = &dockerRegistryOptions{}
+	}
+	if destDockerRegistry == nil {
+		destDockerRegistry = &dockerRegistryOptions{}
+	}
+	srcContext := srcDockerRegistry.getSystemContext(signaturePolicyPath)
+	destContext := destDockerRegistry.getSystemContext(signaturePolicyPath)
+	return &cp.Options{
+		RemoveSignatures: signing.RemoveSignatures,
+		SignBy:           signing.SignBy,
+		ReportWriter:     reportWriter,
+		SourceCtx:        srcContext,
+		DestinationCtx:   destContext,
+	}
+}
+
+func getPolicyContext() (*signature.PolicyContext, error) {
+	var systemContext types.SystemContext
+	policy, err := signature.DefaultPolicy(&systemContext)
```

Comment:
containers/image/copy.Image() requires a policyContext to be passed in, but the SignaturePolicyPath is passed in as well as a part of the copyOptions struct.  Should it stil be set in the PolicyContext or is that redundant?  I left things how Buildah handles it but if it's better to set it in both places I can do that